packages/server/graphql/intranetSchema/resolvers/ping.ts

```diff
@@ -0,0 +1,7 @@
+import {QueryResolvers} from '../resolvers/types'
+
+export interface PingPayload {}
```

Comment:
not the greatest example, but say we wanted to just return `{taskId}`, we could define that here